### DISCUSSION TITLE
Error 132 in docker on Pi0W - will doing a manual build resolve this?

### DISCUSSION THREAD

**Discussion by lits101** [wimaha/TeslaBleHttpProxy#48] [General]
I'm trying to spin up an instance of this in docker on a Raspberry Pi Zero W, but I'm getting error 132. 

Will doing a manual build as per [the instructions](https://github.com/wimaha/TeslaBleHttpProxy?tab=readme-ov-file#build-yourself) work around that error? Or is there a compatibility issue with the Pi Zero W's altogether?

**Comment by lits101** [comment]
I threw some time at this today and I can answer my own question. 

Yes, if you git clone the repo and build manually in golang, then you do work around the Error 132 experienced on the Raspberry Pi Zero W. 

I'm up and running with my own locally hosted BLE proxy. Thanks @wimaha and team for all your work developing this!

**Comment by wimaha** [comment]
I'm trying to add a docker image for raspberry pi zero, but I can't find why it is not working.

**Reply by lits101** [reply]
Unfortunately my logs were empty so I couldn't provide them to help you narrow down the cause. It really wasn't that hard to get it up and running manually though - the hardest part was getting golang updated to the right version.

**Comment by lits101** [comment]
I see a new version has been released. I assume updating is as simple as git cloning the repo into the same directory and then running 'go build .' again?

**Comment by JasonFaulks** [comment]
I'm a total newb here with pi and docker and as luck would have it, trying to get this running on a Pi0W. It doesn't work (well, continual restarts with (132), nothing in the logs) and I'm concerned at having to undertake a workaround not only to get it working but also to update it. Did you ever solve this issue @wimaha? I deleted my image and pulled down the latest but no joy.

**Comment by wimaha** [comment]
There is a new version on dev tag. Can some one test this with Raspberry Pi Zero W?

`image: wimaha/tesla-ble-http-proxy:dev`

**Comment by wimaha** [comment]
I think this is resolved. Please reopen if there are any problems.
